tests/test_detect_target.py

```diff
@@ -47,98 +69,91 @@ def image_zidane():
     """
     Load Zidane image.
     """
-    image = cv2.imread(IMAGE_ZIDANE_PATH)
+    image = cv2.imread(str(IMAGE_ZIDANE_PATH))
     result, zidane_image = image_and_time.ImageAndTime.create(image)
     assert result
     assert zidane_image is not None
     yield zidane_image
 
+@pytest.fixture()
+def expected_bus():
+    """
+    Load expected bus bounding box.
+    """
+    expected_bus = np.loadtxt(BOUNDING_BOX_BUS_PATH)
+    detections = detections_and_time.DetectionsAndTime(0)
+    for i in range(0, expected_bus.shape[0]):
+        result, detection = detections_and_time.Detection.create(expected_bus[i][-4:], int(expected_bus[i][1]), expected_bus[i][0])
+        if result:
+            detections.append(detection)
 
-def rmse(actual: np.ndarray,
-         expected: np.ndarray) -> float:
-        """
-        Helper function to compute root mean squared error.
-        """
-        mean_squared_error = np.square(actual - expected).mean()
-
-        return np.sqrt(mean_squared_error)
-
-
-def test_rmse():
-        """
-        Root mean squared error.
-        """
-        # Setup
-        sample_actual = np.array([1, 2, 3, 4, 5])
-        sample_expected = np.array([1.6, 2.5, 2.9, 3, 4.1])
-        EXPECTED_ERROR = np.sqrt(0.486)
-
-        # Run
-        actual_error = rmse(sample_actual, sample_expected)
+    assert detections is not None
+    yield detections
 
-        # Test
-        np.testing.assert_almost_equal(actual_error, EXPECTED_ERROR)
+@pytest.fixture()
+def expected_zidane():
```

Comment:
It's probably worth moving most of the setup code into a separate function that is called by the fixtures (it wasn't before because the image loading code is simple).